inventory/src/main/java/com/yas/inventory/viewmodel/warehouse/WarehousePostVm.java

```diff
@@ -1,8 +1,20 @@
 package com.yas.inventory.viewmodel.warehouse;
 
 import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+import org.hibernate.validator.constraints.Length;
 
-public record WarehousePostVm(String id, @NotBlank String name, String contactName, String phone, String addressLine1, String addressLine2, String city, String zipCode,
-                              Long districtId, Long stateOrProvinceId, Long countryId) {
-
+@Builder
+public record WarehousePostVm(String id,
+                              @NotBlank @Length(max=450) String name,
```

Comment:
Can we use jakarta.validation.constraints.Size for consistency ?
We should define min value to remove the `@NotBlank.` For example: `@Size(min = 1, max = 450)`

Reply:
I agree with you about that.
I changed @Length to @Size